docs/explanations/conventions.rst

```diff
@@ -223,7 +223,7 @@ Variable Name                  Meaning
 enth                           Specific enthalpy of the entire mixture (across all phases)
 flow_comp["H2O"]               Total flow of H2O (across all phases)
 entr_phase["liq"]              Specific entropy of the liquid phase mixture
-conc_phase_comp["liq", "H2O"]  Concentration of H2O in the liquid phase
+conc_mass_phase_comp["liq", "H2O"]  Concentration of H2O in the liquid phase on a mass basis
```

Comment:
As you noted in the associated issue, there are a few others here that need to be updated with a baisis. Would you be able to update the following

`enth` to `enth_mol`
`flow` to `flow_vol`
`entr_phase['Liq']` to `entr_mol_phase['Liq']`

(I chose a variety of bases to demonstrate their uses).